examples/qmldemo/snapcraft.yaml

```diff
@@ -9,14 +9,7 @@ icon: icon.png
 apps:
   qmldemo:
     command: qmlscene demo.qml --
-    uses: [mir]
```

Comment:
unity7 is only for sdoc. IIRC, this is for mir in kiosk mode which will require the future mir interface.